packages/typespec-azure-core/src/linter.ts

```diff
@@ -4,7 +4,6 @@ import { authRequiredRule } from "./rules/auth-required.js";
 import { byosRule } from "./rules/byos.js";
 import { casingRule } from "./rules/casing.js";
 import { compositionOverInheritanceRule } from "./rules/composition-over-inheritance.js";
-import { friendlyNameRule } from "./rules/friendly-name.js";
```

Comment:
why remove this one?

Reply:
If we do want to remove it, we should remove the associated diagnostic and detection code as well, shouldn't we?

Reply:
Oh weird It must mean it wasn’t added to the rules array

Reply:
yeah that rule was never enabled. But I thought that we used to fail if you try to enable a rule thats not registered, will have to check that